ts/utils/accessibility.ts

```diff
@@ -62,5 +62,13 @@ export const isScreenReaderEnabled = async (): Promise<boolean> => {
 };
 
 // return a string representing the date in a readable format
-export const dateToAccessibilityReadbleFormat = (date: Date) =>
-  `${format(date, I18n.t("global.accessibility.date_format"))}`;
+export const dateToAccessibilityReadbleFormat = (
+  date: Date,
+  readTime: boolean = false
```

Comment:
```suggestion
  format: string = I18n.t("global.accessibility.dateTime_format")
```
I guess this could be more flexible (whatever formats in place of 2)

Reply:
Yep! Sounds good!

Reply:
@Undermaken fixes in 
267423e